docs/tutorials/use-dicom-web-standard-apis-with-c#.md

```diff
@@ -22,9 +22,14 @@ Once you have deployed an instance of the Medical Imaging Server for DICOM, retr
 1. Search for **App Services** and select your Medical Imaging Server for DICOM App Service.
 1. Copy the **URL** of your App Service.
 
+In your application install the following nuget packages:
```

Comment:
Would it be worth it instead to have a *NuGet.config* and *.csproj* movie inline?

Reply:
Updated to have links to nugets